### ISSUE TITLE
Subscribe to user did change

### ISSUE THREAD

**Issue by kdawgwilk** [launchdarkly/js-client#38]
With other clients it's possible to subscribe to when a user changed so that way we can make a call to update our attributes from our server

**Comment by apucacao** [comment]
Hi @kdawgwilk ,

I'm not sure what you mean. Could you explain what you are trying to do?

Cheers,
Alexis

**Comment by kdawgwilk** [comment]
So we recently realized that LaunchDarkly evaluates toggles locally based on the attributes that the current client provides.  This is a problem for us because we need the attributes synced across all of our apps platforms eg, iOS, Android, and EkeyWeb.  We thought LaunchDarkly did this for us and that we could populate all the attributes from only our server and the clients would get the attributes and the changes made to them automatically.  But now we know this isn't the case.  In order to make sure all the client's attributes are in sync we would need to sync down the current attributes from our server but then need a way to know if they have been changed so we don't have to keep polling the server for changes.  In the LaunchDarkly iOS client SDK it has this [`UserUpdatedNotification`](https://github.com/launchdarkly/ios-client/blob/7d70fd0abe52b3ac33ab09e5ec2db674c109fe87/Darkly/DarklyConstants.m#L22) that would be a perfect way to trigger a sync of the new attributes but don't see a way to register for this same type of notification in the JS SDK
